include/solvers/tracer.h

```diff
@@ -328,6 +328,7 @@ class Tracer : public AuxiliaryPhysics<dim, TrilinosWrappers::MPI::Vector>
   // Tracer statistics table
   TableHandler statistics_table;
 
+  // Enable DCDD shock capturing scheme
   const bool DCDD = true;
```

Comment:
so you'll want to implement it in tracer.cc as well?

Reply:
It is already implemented in the tracer. This is where I took it. It's been merged in a past PR.